specification/consumption/resource-manager/Microsoft.Consumption/stable/2018-03-31/examples/ReservationRecommendationsList.json

```diff
@@ -0,0 +1,51 @@
+{
+  "parameters": {
+    "api-version": "2018-03-31",
+    "subscriptionId": "00000000-0000-0000-0000-000000000000"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "id": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Consumption/reservationRecommendations/reservationRecommendations1",
+            "name": "reservationRecommendations1",
+            "type": "Microsoft.Consumption/reservationRecommendations",
+            "sku": "Standard_DS1_v2",
+            "location": "northeurope",
+            "properties": {
+              "lookBackPeriod": "Last7Days",
+              "meterId": "00000000-0000-0000-0000-000000000000",
+              "term": "P1Y",
+              "costWithNoReservedInstances": 0.0,
+              "recommendedQuantity": 1,
+              "totalCostWithReservedInstances": 0.0,
+              "netSavings": 4.634521202630137,
+              "firstUsageDate": "2018-03-06",
```

Comment:
The schema defines this as date/time however you've only provided a date; this is causing model validation failures, see the [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/354117637).  This is applicable to other samples as well.

Reply:
Thanks! for sharing this. I have changed the format to Date.